english/swami-vivekananda/teachings-of-swami-vivekananda/quotes.md

```diff
@@ -24,20 +24,55 @@ Try to be moral, try to be brave, try to be sympathising.
 
 
 3. Have faith that you are all, my brave lads, born to do great things! 
-
 Let not the barks of puppies frighten you -- no, 
-
 not even the thunderbolts of heaven -- but stand up and work!
 
 
 
 
 4. Know that every time you feel weak, you not only hurt yourself but also the Cause. 
+   Infinite faith and strength are the only conditions of success.
+
+
+
+
+5. Who is Helping You, Don’t Forget them.  
+   Who is Loving you, Don’t Hate them .   
+   Who is Believing you, Don’t Cheat them.
+
+6. Anything that makes you weak physically, intellectually
+   and spiritually,reject as poison. 
+
+7. Talk to yourself at  least  once in a Day, Otherwise you may,
+   miss a meeting with an EXCELLENT person in this World.
+
+8. Relationships are more important than life,
+   but it is important for those relationships to have life in them. 
+
+
+9. You have to grow from the inside out. 
+   None can teach you, none can make you spiritual. 
+   There is no other teacher but your own soul.
+
+10. Like me or Hate me,both are in my favor,
+	If you like me I am in your Heart,
+	If you hate me I am in your mind. 
+
+11. BY the study of different religions we find that in essence they are one. 
+
+12. Where can we go to find God if we cannot see Him.
+    In our own hearts and in every living being.
+
+
+13. Fill the brain with high thoughts, highest ideals place them
+	day and night before you and out of that  will come great work.
 
-Infinite faith and strength are the only conditions of success.
+14. Where can we go to find God if we cannot see Him ,
+    In our own hearts and in every living being.
 
 
 
 [Read More](http://www.awakeningindia.org/life/Teachings.aspx)
+[Read More](http://rishikajain.com/2014/01/11/top-ten-famous-quotes-of-swami-vivekananda/)

```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.